frontend/src/routes/about-us/AboutUsPage.vue

```diff
@@ -77,9 +77,7 @@ import LogoComponent from '@/components/design-system/style/LogoComponent.vue';
           formulace otázek klíčovým místem tvorby kalkulačky. Z důvodu
           tranpsarentnosti zde uvádíme náš dokument, kterým se při tvorbě otázek
           řídíme:
-          <a
-            href="https://docs.google.com/document/u/2/d/1JRCRWKQKCHnTCPfzSeeHsSAdEkHMga-vwpppYThA1j4/pub"
-          >
+          <a href="/metodika-tvorby-otazek">
```

Comment:
Tady je třeba použít router link.
```suggestion
          <router-link to="/metodika-tvorby-otazek">
```